lsp/app/aws-lsp-cloudformation-binary/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@lsp-placeholder/aws-lsp-cloudformation-binary",
+    "version": "0.0.1",
+    "description": "CodeBuild CloudFormation Language Server Binary",
+    "main": "out/index.js",
+    "bin": {
+        "awsdocuments-language-server": "./out/index.js"
+    },
+    "scripts": {
+        "compile": "tsc --build",
+        "package-x64": "pkg --targets node18-linux-x64,node18-win-x64,node18-macos-x64 --out-path bin --compress GZip ."
```

Comment:
why isn't this just a task in the main `aws-lsp-cloudformation` projec? why another project needed?

Reply:
sure, those differences can be captured as a build task that produces a separate target. why is a separate project needed? this is a lot of fragmentation, which makes code sharing harder (already presenting in the form of copy-pasted tsconfig.json, etc)